server/database/dbConnection.js

```diff
@@ -0,0 +1,13 @@
+const { Pool } = require('pg');
+require('env2')('./config.env');
+
+let { ELECTION_DAY_DB_URL } = process.env;
+
+if (!ELECTION_DAY_DB_URL)
+  throw new Error('Enviroment variable TRADING_DB_URL must be set');
+
+const options = {
+  connectionString: ELECTION_DAY_DB_URL
+};
+
+module.exports = new Pool(options);
```

Comment:
Just as a note, if you don't add anything to your options object, it'll be cleaner if you put the options in the Pool.
As in `module.exports = new Pool({connectionString: ELECTION_DAY_DB_URL});`

Reply:
Done :+1: